packages/bits/demo/src/components/demo/wizard-v2/wizard-docs/wizard-docs.component.html

```diff
@@ -9,6 +9,14 @@ <h3>Horizontal Layout Wizard</h3>
     <nui-wizard-horizontal-example></nui-wizard-horizontal-example>
 </nui-example-wrapper>
 
+<h3>Wizard with confirmation dialog</h3>
+<p>
+    You can add confirmation dialog when user clicks on the cancel button.
```

Comment:
```suggestion
    It may be a good idea to ask a user whether they really want to quit a wizard, e.g.if they decide to hit `Cancel` while some of the wizard step fields are already filled in. This example reflects this use case and shows how one can add a confirmation dialog to make a user decide whether they really want to quit or stay where they are and continue. 
```